packages/vue-instantsearch/src/components/SearchInput.vue

```diff
@@ -68,8 +68,7 @@
           aria-hidden="true"
           role="img"
           xmlns="http://www.w3.org/2000/svg"
-          width="1em"
-          height="1em"
+          style="width: 1em; height: 1em"
```

Comment:
We could reuse the unitless dimensions from InstantSearch.js reset icon template.

```suggestion
          width="10"
          height="10"
```